verification/curator-service/ui/src/components/LinelistTable.tsx

```diff
@@ -282,11 +301,40 @@ function RowMenu(props: {
                         <span className={classes.menuItemTitle}>Edit</span>
                     </MenuItem>
                 </Link>
-                <MenuItem onClick={handleDelete}>
+                <MenuItem onClick={openDeleteDialog}>
                     <DeleteIcon />
                     <span className={classes.menuItemTitle}>Delete</span>
                 </MenuItem>
             </Menu>
+            <Dialog
+                open={deleteDialogOpen}
+                onClose={(): void => setDeleteDialogOpen(false)}
+                onClick={(e): void => e.stopPropagation()}
+            >
+                <DialogTitle>
+                    Are you sure you want to delete this case?
+                </DialogTitle>
+                <DialogContent>
+                    <DialogContentText>
+                        Case {props.rowId} will be permanently deleted and can
+                        not be recovered.
+                    </DialogContentText>
+                </DialogContent>
+                <DialogActions>
+                    <Button
+                        onClick={(): void => {
+                            setDeleteDialogOpen(false);
+                        }}
+                        color="primary"
```

Comment:
your screenshots didn't seem to have any color, is this "primary" working?

Reply:
Yes, perhaps hard to tell in the screenshot but the text on the buttons is green